common/mariadb-galera/helm/templates/statefulset-proxysql.yaml

```diff
@@ -107,6 +107,31 @@ spec:
         securityContext:
           privileged: {{ $.Values.initContainers.increaseMapCount.securityContext.privileged | default true }}
           runAsUser: {{ $.Values.initContainers.increaseMapCount.securityContext.runAsUser | default 0 | int }}
+      - name: fix-proxysql-fs-permissions
```

Comment:
Please make that initContainer optional like we have done it for the [clean-os-cache](https://github.com/sapcc/helm-charts/blob/8188e613e78536078c381ad3f4b04daa9c55494b/common/mariadb-galera/helm/templates/statefulset-mariadb.yaml#L106) container. Please also create a [proxy.proxysql.fixFsPermissions](https://github.com/sapcc/helm-charts/blob/8188e613e78536078c381ad3f4b04daa9c55494b/common/mariadb-galera/helm/values.yaml#L693) boolean option (default false) and add some documentation similar to [wipeDataAndLog](https://github.com/sapcc/helm-charts/blob/8188e613e78536078c381ad3f4b04daa9c55494b/common/mariadb-galera/helm/values.yaml#L69-L71)